src/releasers/rust.ts

```diff
@@ -57,10 +51,10 @@ export class Rust extends ReleasePR {
 
     const changelogEntry: string = await cc.generateChangelogEntry({
       version: candidate.version,
-      // those need to be prefixed so that the links generated by
-      // the changelog generator go to the proper tags
-      currentTag: addPrefix(`v${candidate.version}`),
-      previousTag: addPrefix(candidate.previousTag),
+      currentTag: await this.normalizeTagName(candidate.version),
+      previousTag: candidate.previousTag
+        ? await this.normalizeTagName(candidate.previousTag)
+        : undefined,
```

Comment:
cc @fasterthanlime - I made this fix for node and python and then I discovered you'd already found this discrepancy when you built the rust releaser. I changed it to be uniform with the others. The snapshot didn't change so we should be good.